src/integrationTest/java/uk/gov/hmcts/darts/ws/EventWebServiceTest.java

```diff
@@ -336,7 +333,7 @@ void testRoutesInvalidDailyListPayloadNoDartsAPIProblemBody(
 
     @ParameterizedTest
     @ArgumentsSource(DartsClientProvider.class)
-    void removesLineBreaksFromDailyListXml(
+    void testCppDailyListXmlWithLineBreaks(
```

Comment:
Isn't it better to specifiy what we are testing?

Reply:
I've made it slightly more general, what name do you think @tonymort ?

Reply:
I think if we've had a bug and a test has been written to cover the scenario of the bug it should remain a specific test. So if theres another bug personally I'd create another test named specifically for that bug.

Reply:
Added a specific test as discussed 👍